addon/instance-initializers/flexberry-logging.js

```diff
@@ -151,8 +136,9 @@ export function initialize(applicationInstance) {
     @private
    */
   function _sendLog(levelName, message, formattedMessage) {
-    if (logLevelEnums[levelName] <= flexberryLogLevel) {	//Meggage category logged (lower or equal high flexberryLogLevel)
-      flexberryLogging.flexberryLogger(levelName, message, formattedMessage);
+    let logLevel = flexberryLogging.logLevelEnums[levelName];
+    if (logLevel <= flexberryLogging.flexberryLogLevel) {	//Meggage category logged (lower or equal high flexberryLogLevel)
```

Comment:
Comments have to be on separate lines and with a space after '//' and a dot at the end.